slave-services/src/main/java/org/iqu/slaveservices/entities/Source.java

```diff
@@ -41,9 +41,15 @@ public void setDescription(String description) {
 		this.description = description;
 	}
 
+	// @Override
+	// public String toString() {
+	// return "Source [id=" + id + ", displayName=" + displayName + ",
+	// description=" + description + "]";
+	// }
+
 	@Override
 	public String toString() {
-		return "Source [id=" + id + ", displayName=" + displayName + ", description=" + description + "]";
+		return "{id:" + id + ", displayName:" + displayName + ", description:" + description + "}";
```

Comment:
This class won't be used in the future when the DTOs and the rest entities will be implemented, and thus this method won't be needed.